crates/bones_camera_shake/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "bones_camera_shake"
+version = "0.1.0"
+authors = ["Peter Hebden (Piturnah) <peterhebden6@gmail.com>", "Fish Folk & Spicy Lobster Developers"]
+license = "MIT OR Apache-2.0"
+
+edition = "2021"
+description = "Small crate providing 2D camera shake for bevy_ecs."
+repository = "https://github.com/fishfolk/bones"
+
+[dependencies]
+bevy = { version = "0.9", default_features = false }
+# For now we are pointing at a git rev but this should be rectified when a new release is made.
+# See https://github.com/Razaekel/noise-rs/pull/281.
+noise = { git = "https://github.com/bsurmanski/noise-rs", rev = "5abdde1b819eccc47e74969c15e1b56ae5a055d6" }
+
+[dev-dependencies]
+bevy_core_pipeline = "0.9"
```

Comment:
Is this needed here?

Reply:
This dependency is required for the [top level example](https://github.com/Piturnah/bones/blob/3f87256f22896ece56f9e06a80adb1a3362e9c45/crates/bones_camera_shake/src/lib.rs#L6) or it won't compile as certain bevy features that aren't required for the core functionality are missing. Alternatively we could tell rustc to ignore that example or remove the example.

Reply:
Ah, I figured it was useful for examples, but I didn't see any. I had forgotten about doc examples.

In that case, it makes sense and I think this is good to go.